ci/docker/runtime_functions.sh

```diff
@@ -785,6 +785,14 @@ deploy_docs() {
     popd
 }
 
+# broken_link_checker
+
+broken_link_checker() {
+    set -ex
+    chmod -R 0755 tests/nightly/broken_link_checker_test/
```

Comment:
Please run chmod locally in your git clone and then commit the filepermission changes. Running this command in CI will modify the git state.